client/src/components/NavTabs.scss

```diff
@@ -0,0 +1,36 @@
+.nav-tabs {
+  margin-bottom: 0;
+  padding-left: 0;
+  height: 100%;
+  display: flex;
+
+  &__tab-item {
+    list-style-type: none;
+
+    a {
```

Comment:
We can't assume this will always be an `<a />` tag. We should have actual class names here.